components/eam/bld/configure

```diff
@@ -1493,10 +1494,10 @@ else {
             if ($print>=2) { print "Advected constituents added by $microphys_pkg microphysics: 8$eol"; }
         }
 
-	if ($clubb_do_adv) {
-	    $nadv += 9;
-            if ($print>=2) { print "Advected constituents added by $microphys_pkg microphysics: 8$eol"; }
-	}
+        if ($clubb_do_adv) {
```

Comment:
Please don't mix white space changes in with PRs that have real code changes.

Reply:
Normally I would agree, but this particular line was edited in a fit of rage after realizing that I had spent days chasing a bug that was ultimately caused by misunderstanding the logic in this block. Whoever added it used tabs when the rest of the file used spaces, and the different spacing caused me to misinterpret the if/else logic. So I would prefer to keep this because I consider it a critical clarity improvement.

Reply:
Ok.  Definitely keep it.